tasks/Create-A-wall.md

```diff
@@ -0,0 +1,27 @@
+#Labyrinth: Create a Wall
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Download and install Inkscape 
+- Create a Wall for the Labyrinth maze 
+- Add your wall to labyrinth/tiles/rooms/wall
+- A merged pull request
+
+Expected outcome:
+- Added atleast one wall
+- One pull requset is succesfully merged
```

Comment:
I would add that the wall should be part of the labyrinth. And that issues based on the created pr must be solved.